src/components/Tokens/TokenDetails/PriceChart.tsx

```diff
@@ -42,28 +42,30 @@ export function calculateDelta(start: number, current: number) {
   return (current / start - 1) * 100
 }
 
-export function getDeltaArrow(delta: number | null | undefined) {
+export function getDeltaArrow(delta: number | null | undefined, iconSize: number) {
```

Comment:
should we default to 20?

Reply:
Great idea! Nice to get that by default and less redundancy across the few places we're using it